lib/dynamoid/components.rb

```diff
@@ -23,7 +23,6 @@ module Components
     include ActiveModel::Naming
     include ActiveModel::Observing if defined?(ActiveModel::Observing)
     include ActiveModel::Serializers::JSON
-    include ActiveModel::Serializers::Xml
```

Comment:
commented similarly on another PR also adding rails 5 support, but don't know which will get fixed up first, so adding here too... beleive this should be `include ActiveModel::Serializers::Xml if defined?(ActiveModel::Serializers::Xml)` so people relying on it in old rails versions/adding it back manually via extracted gem can still use it.